generators/ansible-do/templates/ansible/stack/makefile

```diff
@@ -64,6 +64,10 @@ setup.portainer:
 	@printf "\033[0;3ansible/2m>>> Set up portainer\033[0m\n"
 	ansible-playbook plays/portainer.yml
 
+setup.swarm:
+	@printf "\033[0;3ansible/2m>>> Set up docker swarm\033[0m\n"
+	ansible-playbook plays/swarm-init.yml
```

Comment:
the playbook can be named `swarm` since this is the only swarm-related playbook.

Reply:
understood, it will be changed